packages/esm-patient-forms-app/src/clinical-form-action-menu.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Document20 from '@carbon/icons-react/es/document/20';
+import styles from './clinical-form-action-menu.scss';
+import { useTranslation } from 'react-i18next';
+import { useLayoutType } from '@openmrs/esm-framework';
+import { Button } from 'carbon-components-react';
+
+const ClinicalFormActionMenu: React.FC = () => {
```

Comment:
Also this looks like just a button. Why is it called `Menu`?

Reply:
Renamed them to button